packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Disables the AndroidGradlePluginVersion issue ID in all android packages.
```

Comment:
These changelog changes can all be reverted.

Reply:
Done!

Reply:
> These changelog changes can all be reverted.

I think he means all changelogs can be reverted.

Reply:
> > These changelog changes can all be reverted.
> 
> I think he means all changelogs can be reverted.

Ahhh, my mistake. All changelog changes have been reverted!